x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart.js

```diff
@@ -90,10 +93,27 @@ const anomalyGrayScale = d3.scale
   .domain([3, 25, 50, 75, 100])
   .range(['#dce7ed', '#b0c5d6', '#b1a34e', '#b17f4e', '#c88686']);
 
-function getSvgHeight(showAnnotations) {
+function getChartHeights(height) {
+  const actualHeight = height < minSvgHeight ? minSvgHeight : height;
+  const focusChartHeight = Math.round(actualHeight * percentFocusChartHeight);
+
+  const heights = {
+    focusChartHeight,
+    focusHeight: focusZoomPanelHeight + focusChartHeight,
+  };
+  return heights;
+}
+
+function getSvgHeight(showAnnotations, incomingHeight) {
   const adjustedAnnotationHeight = showAnnotations ? annotationHeight : 0;
+  const incomingHeightActual =
```

Comment:
Nit: Could be `const incomingHeightActual = Math.max(incomingHeight || 0, minSvgHeight);`.